docs/docs/installation.md

```diff
@@ -117,34 +122,28 @@ and compile it.
  mv LL latlong_wgs84
 ```
 
-4.  Actinia Core must be run within a virtual python3 environment:
+4.  Actinia Core & GRASS GIS 7.8 requires a Python 3 environment for running, so we
+    create one (not needed if only Python 3 is present on the machine).
```

Comment:
It is often preferred to create a virtual python environment to also keep dependencies independent from other projects and global installation. Not only because of different python version, so please remove this part. (I know that you just moved this part from below but once you are already on it ;) )
```suggestion
    create one.
```